src/core/event.js

```diff
@@ -264,6 +264,7 @@ Candy.Core.Event = (function(self, Strophe, $) {
 		},
 
 		_addRosterItem: function(item) {
+			item.inRoster = true;
```

Comment:
The `item` here comes directly from a callback on `Candy.Core.getConnection().roster`, so I'm making an assumption that if it's coming all the way to `_addRosterItem()` that it's for someone who can be safely said to be in our roster. It can get here either through it's sibling `_addRosterItems()` or through a direct callback created in `Candy.Core.Action.Jabber.Roster()`.